From 522c0e6f6b22e341ca73161fca98efda54550413 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 06 Sep 2018 10:15:23 +0300
Subject: [PATCH] Makefile: fix distclean for hl-fw-loader

The fw_loader has been renamed to hl-fw-loader, but for some reason
the distclean target has not been updated.
Update it now.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index 6659957..9cd20f6 100644
--- a/Makefile
+++ b/Makefile
@@ -190,7 +190,7 @@
 		uboot-custom linux-custom armcp-custom \
 		spdlog-custom specs-custom specs_external-custom \
 		host-uboot-test-hooks-custom host-tbot-custom \
-		thermal-custom preboot-custom host-fw_loader-custom \
+		thermal-custom preboot-custom host-hl-fw-loader-custom \
 		btl-custom; \
 	fi
 
